### DISCUSSION TITLE
Document listing type document link

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [facebook/docusaurus#8014] [Q&A] [answered]
Is there any way to link to a document, so it looks like the folder indexes?

I would like to use these boxes (representing directories)
![grafik](https://user-images.githubusercontent.com/50182241/187032680-3131d775-4259-40ee-91c7-9e949a96bddd.png)
in a document where i have other Text.  

Is that possible?

**Comment by slorber** [comment, marked as answer]
> Is there any way to link to a document, so it looks like the folder indexes?

I have no idea what you means here

---

Not sure to 100% understand what you mean here, but I think it it already possible in the upcoming Docusaurus 3.x canaries:
https://docusaurus.io/docs/next/sidebar/items#generated-index-page

![CleanShot 2022-08-31 at 14 45 02@2x](https://user-images.githubusercontent.com/749374/187681332-be242b8d-6668-42a7-a16a-d2610e2d922e.png)

PR will be backported in v2.1 soon (https://github.com/facebook/docusaurus/pull/8008)

---

Note you can also provide your own list of items as props:

```jsx
<DocCardList items={...}/>
```

By default, we display what's inside the parent category as a convenient shortcut

**Reply by ezbeazy** [reply]
```js
// /docs/category/index.mdx

import DocCardList from '@theme/DocCardList';

<DocCardList />
```

I get the following error:

Page Crashed
Cannot read properties of undefined (reading 'filter')

Does this not work when Autogenerating the Sidebar?

**Reply by ezbeazy** [reply]
Didn't realize this was a new feature for 2.1.0

fixed with updating from from 2.0.1 to 2.1.0
